.travis.yml

```diff
@@ -56,17 +59,14 @@ install:
 
   # Install Drush.
   - export PATH=$PATH:/home/travis/.composer/vendor/bin
-#  - composer selfupdate
+  - composer selfupdate
   - composer global require drush/drush:dev-master
 
 before_script:
-  # Add always_populate_raw_post_data=-1 to php.ini.
-  - echo "always_populate_raw_post_data=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-
   # Checkout a directory structure with all dependencies.
   - git clone --depth 1 --branch 8.0.x http://git.drupal.org/project/drupal.git $TRAVIS_BUILD_DIR/../drupal
   - git clone --branch 8.x-1.x https://github.com/dickolsson/drupal-key_value.git $TRAVIS_BUILD_DIR/../drupal/modules/key_value
-  - git clone --branch 8.x-1.x https://github.com/dickolsson/drupal-multiversion.git $TRAVIS_BUILD_DIR/../drupal/modules/multiversion
+  - git clone --branch new-revision-creation-fixes https://github.com/dickolsson/drupal-multiversion.git $TRAVIS_BUILD_DIR/../drupal/modules/multiversion
```

Comment:
@jeqq Can you please open a new multiversion PR for this branch so that I can start reviewing it?